automation/src/test/scala/org/broadinstitute/dsde/test/api/SnapshotAPISpec.scala

```diff
@@ -0,0 +1,143 @@
+package org.broadinstitute.dsde.test.api
+
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
+import akka.http.scaladsl.model.{StatusCodes, Uri}
+import akka.http.scaladsl.model.Uri.Path
+import bio.terra.datarepo.api.RepositoryApi
+import bio.terra.datarepo.client.ApiClient
+import bio.terra.workspace.model.{DataReferenceList, ReferenceTypeEnum}
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.workbench.auth.AuthToken
+import org.broadinstitute.dsde.workbench.config.ServiceTestConfig.FireCloud
+import org.broadinstitute.dsde.workbench.config.UserPool
+import org.broadinstitute.dsde.workbench.fixture.{BillingFixtures, WorkspaceFixtures}
+import org.broadinstitute.dsde.workbench.service.Rawls
+import org.broadinstitute.dsde.workbench.service.util.Tags
+import org.scalatest.freespec.AnyFreeSpecLike
+import org.scalatest.matchers.should.Matchers
+
+import java.util.UUID
+import scala.collection.JavaConverters._
+import scala.util.{Failure, Success, Try}
+
+class SnapshotAPISpec extends AnyFreeSpecLike with Matchers
+  with WorkspaceFixtures with BillingFixtures
+  with SprayJsonSupport with LazyLogging {
+
+  private val dataRepoBaseUrl = FireCloud.dataRepoApiUrl
+
+  // assume we have a valid dataRepoBaseUrl. We've had bugs/problems in our config so this is a safeguard.
+  assume(Try(Uri(dataRepoBaseUrl)).isSuccess,
```

Comment:
Wouldn't we want this to fail if we had bugs or problems in our configs?

Reply:
`assume`  will cancel the test - I thought that would be less disruptive. Would you rather fail it so we notice? We can do that!

Reply:
I think if there are other ways of telling if our configs are bad, then this might be redundant. But if there aren't any other ways of quickly telling, then it might be useful for the tests to fail!

Reply:
@ansingh7115 I am now aborting the entire suite (which shows up as a failure) via a `beforeAll` ... is that better?

Reply:
sure, that works - thank you!